src/test_workflow/integ_test/integ_test_start_properties_opensearch.py

```diff
@@ -9,6 +9,8 @@
 
 
 class IntegTestStartPropertiesOpenSearch(IntegTestStartProperties):
-    def __init__(self, path):
+    dependency_installer: DependencyInstallerOpenSearch
+
+    def __init__(self, path: str):
```

Comment:
This method should also return `None`?

Reply:
Yeah, so why isn’t mypi failing on this? Any ideas?

Reply:
Found the answer in https://mypy.readthedocs.io/en/stable/common_issues.html

[__init__](https://docs.python.org/3/reference/datamodel.html#object.__init__) method has no annotated arguments or return type annotation. [__init__](https://docs.python.org/3/reference/datamodel.html#object.__init__) is considered fully-annotated if at least one argument is annotated, while mypy will infer the return type as None.

To be consistent, added missing annotations in [Added missing __init__ annotations.](https://github.com/opensearch-project/opensearch-build/pull/2123/commits/a8832cb9201f2088f9dab08fbce6f0a2cffa212b)

Reply:
I was wondering the same thing. Thanks for the explanation!